scripts/pipeline/orthology_benchmark.py

```diff
@@ -157,12 +165,57 @@ def get_core_names(species_names: List[str], host: str, port: int, user: str) ->
     return core_names
 
 
-def prep_input_for_orth_tools():
-    """Docstring"""
+def prep_input_for_orth_tools(source_dir: str, target_dir: str) -> None:
+    """Prepares input files for selected orthology inference tools.
 
+    Creates symlinks to input fasta files in `target_dir`.
 
-def run_orthology_tools():
-    """Docstring"""
+    Args:
+        source_dir: Path to the directory containing .fasta files.
+        target_dir: Path to the directory where symlinks to .fasta files will be created.
+
+    Raises:
+        RuntimeError: If command to create symlinks fails for any reason.
+    """
+    # OrthoFinder
+    script_symlinks = os.path.join(Path(__file__).parents[2], "scripts", "pipeline", "symlink_fasta.py")
+
+    try:
+        subprocess.run([script_symlinks, "-d", source_dir, "-s", target_dir],
+                       capture_output=True, check=True, text=True)
+    except subprocess.CalledProcessError as exc:
+        msg = f"Command '{exc.cmd}' returned non-zero exit status {exc.returncode}"
+        if exc.stdout:
+            msg += f"\n  StdOut: {exc.stdout}"
+        if exc.stderr:
+            msg += f"\n  StdErr: {exc.stderr}"
+        raise RuntimeError(msg) from exc
+
+
+def run_orthology_tools(input_dir: str, orthofinder_params: str) -> None:
```

Comment:
My only comment here is, since you have made changes suggested by David, I want to see which direction this is headed now: with all the orthofinder tool params wrapped in  a single param string how will it look with different orthology tools? Will each orthology tool require it's own `orthotool_params` ? Perhaps they could be provided in that more tool-flexible way since they are now a string? You would include in the end of the string the flag/param to include the input once processed, and otherwise, pass in tool and params in the same blackbox style we have been looking at - so no tool has to be hard-coded in the code as such - with another param/flag to determine the type of preprocessing required in the file prep. This might be a bit of a pain if you need this code quite soon, so I don't think I expect the changes this increment, but something to think about for the next.

Reply:
Actually, one more comment, perhaps pulling in the most recent changes from `main` will help Travis get over his grievances. My approval is conditional on Travis passing, as one would expect :crossed_fingers:

Reply:
I do not understand your comment, could you please detail more because it seems interesting?

Reply:
Thank you @CristiGuijarro for a speedy reaction and your thoughts on the future developments. The current plan is to move on with pipeline development by adding functionality to prepare input files for GOC score calculation (GTF files, files with OrthoFinder putative orthologs) and calculate GOC scores. Once that's done, i.e. I have a fully working pipeline, I'd like to move away from providing pipeline parameters via command line. All pipeline params would be provided in a (`json`) file. There I could also have a full path to `orthofinder_exe`. In case I add another tool to the pipeline before that, it would be dealt in the same way as OrthoFinder now: the pipeline would run the tool in its default mode if no additional params are provided, and if there are additional params, they'd be provided by corresponding `orthotool_params` (a user would specify them via string in a command line).

How does that sound to you?

P.S. The pipeline is conceived to be running all tools specified in `run_orthology_tools()`. It'd be nice to be able to run the pipeline only for a specified subset of orthology tools. I think that could be another small ticket. But let's make one step at a time 🙂